mlflow/tracking/fluent.py

```diff
@@ -284,6 +405,17 @@ def log_params(params):
     :param params: Dictionary of param_name: String -> value: (String, but will be string-ified if
                    not)
     :returns: None
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
+
+        params = {"learning_rate": 0.01, "n_estimators": 10}
+
+        # Use context manager to create a new run and log a batch of parameters
+        with mlflow.start_run(run_name="My Runs"):
```

Comment:
```suggestion
        with mlflow.start_run():
```

To make the code as simple as possible.

Reply:
Removed all to make it simple.